models/user.js

```diff
@@ -1,23 +1,19 @@
 var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
-var valuesSchema = mongoose.Schema({
-  interests: [String]
-});
-
-var mechanicsSchema = mongoose.Schema({
-  cleanliness: Number,
-  noise: Number,
-  pets: Boolean,
-  sleep: String
-});
-
 var userSchema = mongoose.Schema({
   name: String,
   email: String,
   about: String,
-  values: valuesSchema,
-  mechanics: mechanicsSchema 
+  values: {
+    interests: [String]
+  },
+  mechanics: {
+    cleanliness: Number,
+    noise: Number,
+    pets: Boolean,
+    sleep: String
+  }
```

Comment:
Nesting mongo schemas can only happen with Refs now, and refs would usually just have an ObjectId used as reference to a different model. Since I assume all users will have their own mechanics and their own values, it doesn't make sense to have a different schema for it, just some structure. It's not like we're going to come up with a bunch of preset values and mechanics.